torchio/transforms/augmentation/intensity/random_labels_to_image.py

```diff
@@ -51,38 +62,39 @@ class RandomLabelsToImage(RandomTransform, IntensityTransform):
 
     Example:
         >>> import torchio
-        >>> from torchio import RandomLabelsToImage, DATA, RescaleIntensity, Compose
-        >>> from torchio.datasets import Colin27
-        >>> colin = Colin27(2008)
-        >>> # Using the default gaussian_parameters
-        >>> transform = RandomLabelsToImage(label_key='cls')
-        >>> # Using custom gaussian_parameters
-        >>> label_values = colin['cls'][DATA].unique().round().long()
-        >>> gaussian_parameters = {
-        ...     label: {
-        ...         'mean': i / len(label_values),
-        ...         'std': 0.01
-        ...     }
-        ...     for i, label in enumerate(label_values)
-        ... }
-        >>> transform = RandomLabelsToImage(label_key='cls', gaussian_parameters=gaussian_parameters)
-        >>> transformed = transform(colin)  # colin has a new key 'image' with the simulated image
+        >>> from torchio import RandomLabelsToImage, RescaleIntensity, RandomBlur, Compose
+        >>> from torchio.datasets import ICBM2009CNonlinearSymmetryc
+        >>> sample = ICBM2009CNonlinearSymmetryc()
+        >>> # Using the default parameters
+        >>> transform = RandomLabelsToImage(label_key='tissues')
+        >>> # Using custom mean and std
+        >>> transform = RandomLabelsToImage(
+        ...     label_key='tissues', mean=[0.33, 0.66, 1.], std=[0, 0, 0]
+        ... )
+        >>> # Discretizing the partial volume maps and blurring the result
+        >>> simulation_transform = RandomLabelsToImage(
+        ...     label_key='tissues', mean=[0.33, 0.66, 1.], std=[0, 0, 0], discretize=True
+        ... )
+        >>> blurring_transform = RandomBlur(std=0.3)
+        >>> transform = Compose([simulation_transform, blurring_transform])
+        >>> transformed = transform(sample)  # sample has a new key 'image' with the simulated image
         >>> # Filling holes of the simulated image with the original T1 image
         >>> rescale_transform = RescaleIntensity((0, 1), (1, 99))   # Rescale intensity before filling holes
         >>> simulation_transform = RandomLabelsToImage(
-        ...     label_key='cls',
+        ...     label_key='tissues',
         ...     image_key='t1',
-        ...     gaussian_parameters={0: {'mean': 0, 'std': 0}}
+        ...     used_labels=[0, 1]
         ... )
         >>> transform = Compose([rescale_transform, simulation_transform])
-        >>> transformed = transform(colin)  # colin's key 't1' has been replaced with the simulated image
+        >>> transformed = transform(sample)  # sample's key 't1' has been replaced with the simulated image
     """
     def __init__(
             self,
-            label_key: Optional[str] = None,
-            pv_label_keys: Optional[Sequence[str]] = None,
+            label_key: str,
+            used_labels: Optional[Sequence[int]] = None,
             image_key: str = 'image',
```

Comment:
Can we use something less likely to have already been used by the user? For example, `image_from_labels`.